test/language/class/class-subclass-binding.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class subclass binding
+---*/
+class Base {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+  }
+}
+
+var obj = {};
+class Subclass extends Base {
+  constructor(x, y) {
+    super(x,y);
+    assert.sameValue(this !== obj, true);
```

Comment:
Based on [recent conversation](https://github.com/tc39/test262/issues/173#issuecomment-75316654), it sounds like we can simply transform `assertTrue` to `assert`, regardless of arguments.